### PR TITLE
Ensure in root directory before publish

### PR DESCRIPTION
**Pull request by hugoesthere**
Ensure user is in root directory before publishing, otherwise all the commands will be run in the wrong subdirectories.

@kevgo

### CODE REVIEW

File: bin/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "inquirer": "3.0.1",
     "livescript": "1.5.0",
     "o-tools-livescript": "1.2.3",
+    "path": "0.12.7",
```
Comment (kevgo) [inline bin/package.json]: You should be able to use the built-in [path](https://nodejs.org/api/path.html#path_path_basename_path_ext) module in Node here.

File: bin/publish [hunk 2]
```diff
@@ -193,6 +195,10 @@ function ensure-on-master
   if current-branch.trim! isnt 'master' then console.log red 'You must be on the master branch to publish' ; process.exit 1
 
 
+function ensure-in-root-dir
+  if path.basename(process.cwd!) isnt 'exosphere' then console.log red 'You must be in the root directory to publish' ; process.exit 1
```
Comment (kevgo) [inline bin/publish]: Remove `then` and move the code after it to the next line, indented

**Comment by hugoesthere** [comment]
@kevgo comments addressed

**Review by kevgo — APPROVED** [review]
Alright!

**Pull request closed without merging**